File: pset 1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Enter Height:");
    }
    while (height < 1 || height > 8);

    for (int i = 0; i < height; i++)
    {
        for (int m = 1; m < height - i; m++)
        {
            printf(" ");
        }

        for (int z = 0; z <= i; z++)
        {
            printf("#");
        }

        printf("  ");

        for (int y = 0; y <= i; y++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: pset 6/sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    while True:
        height = get_int("Height: ")
        if height >= 1 and height <= 8:
            break
        else:
            continue
    make_blocks(height)


def make_blocks(height):
    for a in range(height):
        print(" " * (height - (a + 1)), end="")
        print("#" * (a + 1) + "  " + "#" * (a + 1))


main()
